console/web3ext/web3ext.go

```diff
@@ -1394,6 +1400,14 @@ web3._extend({
 			params: 1
 		}),
 		new web3._extend.Method({
+			name: 'getBridgePairByAlias',
+			call: 'subbridge_getBridgePairByAlias',
+			params: 1
+		}),
+		new web3._extend.Method({
+			name: 'changeBridgeAlias',
+			call: 'subbridge_changeBridgeAlias',
+			params: 2
```

Comment:
```suggestion
			params: 2
		new web3._extend.Method({
```

Reply:
:( I should have checked the final commits out. Fixed. Thank you.